versions-manifest.json

```diff
@@ -3888,6 +4122,54 @@
       }
     ]
   },
+  {
+    "version": "3.9.20",
+    "stable": true,
+    "release_url": "https://github.com/actions/python-versions/releases/tag/3.9.20-10765735733",
+    "files": [
+      {
+        "filename": "python-3.9.20-darwin-x64.tar.gz",
+        "arch": "x64",
+        "platform": "darwin",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-darwin-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.9.20-linux-20.04-x64.tar.gz",
+        "arch": "x64",
+        "platform": "linux",
+        "platform_version": "20.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-linux-20.04-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.9.20-linux-22.04-arm64.tar.gz",
+        "arch": "arm64",
+        "platform": "linux",
+        "platform_version": "22.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-linux-22.04-arm64.tar.gz"
+      },
+      {
+        "filename": "python-3.9.20-linux-22.04-x64.tar.gz",
+        "arch": "x64",
+        "platform": "linux",
+        "platform_version": "22.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-linux-22.04-x64.tar.gz"
+      },
+      {
+        "filename": "python-3.9.20-linux-24.04-arm64.tar.gz",
+        "arch": "arm64",
+        "platform": "linux",
+        "platform_version": "24.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-linux-24.04-arm64.tar.gz"
+      },
+      {
+        "filename": "python-3.9.20-linux-24.04-x64.tar.gz",
+        "arch": "x64",
+        "platform": "linux",
+        "platform_version": "24.04",
+        "download_url": "https://github.com/actions/python-versions/releases/download/3.9.20-10765735733/python-3.9.20-linux-24.04-x64.tar.gz"
+      }
+    ]
```

Comment:
Is there a reason [3.8.20](https://www.python.org/downloads/release/python-3820/) is missing from this update?